lib/packer/builders/amazon.rb

```diff
@@ -12,7 +12,7 @@ def initialize
           'instance_type',
           'region',
           'source_ami',
-          'ssh_username'
+          ['ssh_username', 'communicator']
```

Comment:
Ah. Gotcha. I see what you mean. Yea, I can make this work. It looks like the communicator stuff should get pulled out in to a module that can get dropped in to multiple builder classes. Shouldn't be too hard!